_posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md

```diff
@@ -0,0 +1,73 @@
+* We’ve launched tools to help students [compare college costs](http://www.consumerfinance.gov/paying-for-college/compare-financial-aid-and-college-cost/) and [determine their repayment options](http://cfpb.github.io/articles/repay-student-debt-feature-revamp/).
+* We’ve designed [standardized disclosures](http://cfpb.github.io/articles/designing-prepaid-card-disclosures/) for prepaid cards to help consumers understand differences in costs and fees.
+* We’ve designed and developed a web-based mapping tool to help our examiners identify redlining and other discriminatory lending practices.
+* We’ve launched [eRegulations](http://www.consumerfinance.gov/eregulations/), a web-based tool that makes regulations easy to find, read and understand.
+
+For the remainder of 2015, we’re focusing on several initiatives:
+
+* We’re improving our overall web presence at [consumerfinance.gov](http://www.consumerfinance.gov/) as a platform for future guides, tools, and services.
+* We’re consolidating our consumer complaint forms into a unified experience with support for complex complaints that involve multiple companies.
+* We’re developing applications to search and analyze data from consumer complaints, comments, and inquiries.
+* We’re creating new types of interactive disclosures to help ensure that consumers know all the facts when they’re making financial decisions.
+
+
+## How we work and what we’re looking for
+
+Our Design & Development Team manages this work by empowering small, cross-disciplinary teams to collaborate remotely and get things done. Our UX designers work alongside graphic designers, front- and back-end developers, data specialists, and subject matter experts every day, with some of them serving as the Scrum Master or Product Owner for their project team. We meet as cross-project “feedback houses” to help spread information among designers, each project team presents their recent work on Friday afternoons, and we use our publicly-available [Design Manual](https://cfpb.github.io/design-manual/) to maintain design and brand patterns across projects. 
+
+The next phase in our growth is hiring an experienced manager to lead our disciplines of interaction design, user research, and content strategy as a mentor, supervisor, technical expert, and visionary advocate for user-centered design. The UX Lead position will have an option for working remotely outside of Washington, DC, though it will require a significant amount of travel to our headquarters as well as other locations where we conduct user research. We want to find the best possible candidate to help our team grow, so we’re conducting a national search.
+
+We hear from hundreds of American consumers every day who believe that we may be able to help them find a solution to problems that are crippling their day-to-day lives. Designers are a part of the solution here. Please join us.
+
+
+
+To learn more about the CFPB’s UX Lead position, email us at [jobs@cfpb.gov](jobs@cfpb.gov).

```

Comment:
Needs to be a `mailto:` link. Also, should we add a prefilled subject string to this link?